Assignment 2/180501135/attendanceStudents.py

```diff
@@ -0,0 +1,10 @@
+s = ["Ram-P", "Malar-A", "Ahuja-P", "Vijay-P", "Harini-A"]
+print('The present students were:')
+b=0
+print("The present students were:")
+for i in s:
+    k=s[b]
+    if k[-1]=='P':
+        print(k[:-2]);
+    b+=1;
+        
```

Comment:
Make the code dynamic. The code should be able to accept any number of inputs. Feel free to include extra input variables like the number of students and make the program dynamic.

Reply:
The Code is made dynamic ( the user can now give the inputs)..